functions/Get-DbaService.ps1

```diff
@@ -146,125 +146,130 @@ function Get-DbaService {
-                            } else {
-                                Add-Member -Force -InputObject $service -MemberType NoteProperty -Name Clustered -Value ''
-                                Add-Member -Force -InputObject $service -MemberType NoteProperty -Name VSName -Value ''
-                            }
+                        $ClusterServiceTypeList = @(1, 2, 5, 7)
+                        if ($ClusterServiceTypeList -contains $service.SQLServiceType) {
+                            Add-Member -Force -InputObject $service -MemberType NoteProperty -Name Clustered -Value ($serviceAdvancedProperties | Where-Object PropertyName -eq 'CLUSTERED' ).PropertyNumValue
+                            Add-Member -Force -InputObject $service -MemberType NoteProperty -Name VSName -Value ($serviceAdvancedProperties | Where-Object PropertyName -eq 'VSNAME' ).PropertyStrValue
+                        } else {
+                            Add-Member -Force -InputObject $service -MemberType NoteProperty -Name Clustered -Value ''
+                            Add-Member -Force -InputObject $service -MemberType NoteProperty -Name VSName -Value ''
                         }
-                        $outputServices += $service
                     }
+                    $outputServices += $service
                 }
-                if ($AdvancedProperties) {
-                    $defaults = "ComputerName", "ServiceName", "ServiceType", "InstanceName", "DisplayName", "StartName", "State", "StartMode", "Version", "SPLevel", "SkuName", "Clustered", "VSName"
-                } else {
-                    $defaults = "ComputerName", "ServiceName", "ServiceType", "InstanceName", "DisplayName", "StartName", "State", "StartMode"
-                }
-                if ($outputServices) {
-                    $outputServices | Select-DefaultView -Property $defaults -TypeName DbaSqlService
-                } else {
-                    Stop-Function -Message "No services found in relevant namespaces on $Computer. Please note that this function is available from SQL 2005 up."
-                }
+            }
+            if ($AdvancedProperties) {
+                $defaults = "ComputerName", "ServiceName", "ServiceType", "InstanceName", "DisplayName", "StartName", "State", "StartMode", "Version", "SPLevel", "SkuName", "Clustered", "VSName"
+            } else {
+                $defaults = "ComputerName", "ServiceName", "ServiceType", "InstanceName", "DisplayName", "StartName", "State", "StartMode"
+            }
+            if ($outputServices) {
+                $outputServices | Select-DefaultView -Property $defaults -TypeName DbaSqlService
             } else {
-                Stop-Function -EnableException $EnableException -Message "Failed to connect to $Computer" -Continue
+                Write-Message -Level Verbose -Message "No services found in relevant namespaces on $computer. Please note that this function is available from SQL 2005 up."
```

Comment:
I believe we can drop the later part of this message now. It is not really relevant that I see.

Reply:
Ok, will do.